README.md

```diff
@@ -35,12 +35,12 @@ Implemented:
 - [elmls](https://github.com/neovim/nvim-lsp#elmls)
 - [gopls](https://github.com/neovim/nvim-lsp#gopls) (has some errors)
 - [pyls](https://github.com/neovim/nvim-lsp#pyls)
+- [rust-analyzer](https://github.com/rust-analyzer/rust-analyzer)
```

Comment:
This should go in README_preamble.md. README.md is autogenerated.

Also it is meant to be a link to the section in the README, not a link to the project. To be honest, I'll probably auto generate this part eventually.